lib/util/http/index.ts

```diff
@@ -25,6 +27,12 @@ import './legacy';
 
 export { RequestError as HttpError };
 
+type JsonArgs<T extends HttpOptions> = {
```

Comment:
needs to be exported, as it's used as arg of a protected method

Reply:
I think this protected method should become private. Don't think it meant to be overrided.

Reply:
Well, seems like I already did it, so the type can be module-local